apps/tutorial_servo_control/src/TutorialServo.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+#include "mbed.h"
+
+class TutorialServo{
+
+  private:
+
+        PwmOut m_servoPwmOut;
+        const float m_servoRangeInDegrees;
+        const float m_minPulsewidthInMs;
+        const float m_maxPulsewidthInMs;
+
+  public:
+
+        TutorialServo(PinName servoPin, float servoRangeInDegrees = 180, float minPulsewidthInMs = 1, float maxPulsewidthInMs = 2);
```

Comment:
not sure if this is just github but I'd prefer to have these lines of code aligned for consistency. Did you run Clang formatting?